simulator/docs/record-and-replay-cluster-events.md

```diff
@@ -0,0 +1,95 @@
+# [Beta] Record your real cluster's events and replay them in the simulator
+
+You can record events from your real cluster and replay them in the simulator. This feature is useful for reproducing issues that occur in your real cluster.
+
+## Record events
+
+To record events from your real cluster, you need to follow these steps:
+
+1. Set `true` to `recorderEnabled`.
+2. Set the path of the kubeconfig file for your cluster to `KubeConfig`.
+   - This feature only requires the read permission for events.
+3. Set the path of the file to save the recorded events to `recordedFilePath`.
+4. Make sure the file path is mounted to the simulator container.
+
+```yaml
+recorderEnabled: true
+kubeConfig: "/path/to/your-cluster-kubeconfig"
+recordedFilePath: "/path/to/recorded-events.json"
+```
+
+```yaml
+volumes:
+  ...
+  - ./path/to/recorded-events.json:/path/to/recorded-events.json
+```
+
+> [!NOTE]
+> When a file already exists at `recordedFilePath`, it backs up the file in the same directory adding a timestamp to the filename and creates a new file for recording.
```

Comment:
Looks confusing. I prefer either just override the file without the error or make the recorder put out the error.

Reply:
I'll fix it to put out the error.